libmscore/breath.cpp

```diff
@@ -64,7 +64,7 @@ void Breath::layout()
       {
       bool palette = (track() == -1);
       if (!palette) {
-            int voiceOffset = track() % 2 * 4 * spatium();
+            int voiceOffset = placeBelow() * 4 * spatium();
```

Comment:
Isn't this "magic" `4` rather `VOICES`?

Reply:
The 4 is to move the breath 4 lines down, from the fifth to the first line.

Reply:
Better to use `(staff()->lines(tick()) - 1)` instead of `4`.

Reply:
very good :-)